src/CommandLineArgumentsParser/CommandLineParser.cs

```diff
@@ -54,6 +56,10 @@ public class CommandLineParser : IDisposable
 
     #endregion
 
+    public CommandLineParser(ILogger<CommandLineParser> logger)
```

Comment:
``` c#
public CommandLineParser(ICommandLineParserLogger? logger)
{
  _logger = logger ?? new ConsoleLogger();
}
```

Reply:
Please see https://github.com/j-maly/CommandLineParser/pull/82#discussion_r945147930 Console logger was just an example. Consumers could be using any sink (file, eventlog etc). Hence this change is not needed.